services/api-server/src/simcore_service_api_server/api/routes/credits.py

```diff
@@ -12,7 +12,7 @@
 @router.get(
     "/price",
     status_code=status.HTTP_200_OK,
-    response_model=GetCreditPrice,
+    response_model=GetCreditPriceLegacy,
```

Comment:
I wonder how is it going to impact the name of the model when it is autogenerated in the client. Keep an eye since probably it will also change.

Reply:
Yep, I will keep an eye on this